server/app/filters/CiviFormProfileFilter.java

```diff
@@ -45,7 +45,6 @@ private boolean profileIsMissing(Http.RequestHeader requestHeader) {
   private boolean shouldApplyThisFilter(Http.RequestHeader requestHeader) {
     return NonUserRoutePrefixes.noneMatch(requestHeader)
         && !requestHeader.path().startsWith("/callback")
-        && requestHeader.method().equals("GET")
```

Comment:
**thought:** Would it be worth adding logging to see when the request header is not GET and the profile is missing? It may make us feel more confident making this change

Reply:
Fair, thanks for the feedback! I've changed it to just add HEAD instead of removing the condition to reduce the risk.

Reply:
SGTM, thanks!